src/skins/blau.tsx

```diff
@@ -171,6 +171,10 @@ export const getBlauSkin: GetKnownSkin = () => {
             tagBackgroundSuccess: palette.blauGreen10,
             tagBackgroundWarning: palette.blauYellow10,
             tagBackgroundError: palette.blauRed10,
+            cardContentOverlay: `linear-gradient(180deg, ${applyAlpha(palette.grey6, 0)} 0%, ${applyAlpha(
+                palette.grey6,
+                0.4
+            )} 30%, ${applyAlpha(palette.grey6, 0.7)} 100%)`,
```

Comment:
this formatting is quite ugly. It should get fixed after upgrading to prettier v3